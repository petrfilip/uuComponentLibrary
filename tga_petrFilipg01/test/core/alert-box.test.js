import UU5 from "uu5g04";
import TgaPetrFilip from "tga_petrFilipg01";

const { shallow } = UU5.Test.Tools;

describe(`TgaPetrFilip.Core.AlertBox`, () => {
  it(`default props`, () => {
    const wrapper = shallow(<TgaPetrFilip.Core.AlertBox />);
    expect(wrapper).toMatchSnapshot();
  });
});
