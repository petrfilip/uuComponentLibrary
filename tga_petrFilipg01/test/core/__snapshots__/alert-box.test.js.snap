// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TgaPetrFilip.Core.AlertBox default props 1`] = `
<Component
  className="tgapetrfilip-core-0"
  colorSchema="info"
>
  <Component>
    <Component
      colWidth="xs-12 s-1"
    >
      <Component
        icon="uu5-finder"
        tooltip="aaaaabbbbbccccc"
      />
    </Component>
    <Component
      colWidth="xs-12 s-10"
    >
      <Component>
        aaaaabbbbbccccc
      </Component>
    </Component>
    <Component
      colWidth="xs-12 s-1"
    >
      <Component
        onClick={[Function]}
      >
        OK
      </Component>
    </Component>
  </Component>
</Component>
`;
